Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -926,28 +926,32 @@ - (CGSize)bodyMediaSizeForContentWidth:(int)contentWidth
             }
 
             return CGSizeRound(CGSizeMake(mediaWidth, mediaHeight));
```

Comment:
why this break after a return? Did you intend to leave this return?

Reply:
Fixed, thanks.